src/cryptoKeyUtil.ts

```diff
@@ -0,0 +1,69 @@
+import { logger } from "@blockr/blockr-logger";
+import { ec as EC } from "elliptic";
+import { injectable } from "inversify";
+import { KeyPairException } from "./exceptions";
+import { SignatureException } from "./exceptions/signature.exception";
+
+@injectable()
+export class CryptoKeyUtil {
+    private ec: EC;
+
+    constructor() {
+        this.ec = new EC("secp256k1");
+    }
+
+    /**
+     * Generate a new (EC) crypto key pair.
+     * @returns {EC.KeyPair | IKeyPair} EC key pair.
+     * @throws {KeyPairException} when the generated key pair fails to verify.
+     */
+    public generateKeyPair(): EC.KeyPair {
+        logger.info("Generating key pair.");
```

Comment:
I personally think that if we're going with the approach of a separate crypto library, this library is not expected to perform any logging on its own. I would rather expect the consuming application to perform logging.

tdlr; should we perform logging within this library?